tijanabekic/assignment4/Assignment 4.java

```diff
@@ -0,0 +1,101 @@
+import static org.junit.Assert.*;
+
+public class A4Q1 {
+
+    public static void findIslandEnd(boolean[][] grid, int i, int j, int row, int column) {
+        if (i < 0 || i >= row || j < 0 || j >= column) {
+            return;
+        }
+        if (grid[i][j]) {
+            //fload this cell and look for the and of the island
+            grid[i][j] = false;
+            findIslandEnd(grid, i - 1, j, row, column); //up
+            findIslandEnd(grid, i + 1, j, row, column); // down
+            findIslandEnd(grid, i, j - 1, row, column); // left
+            findIslandEnd(grid, i, j + 1, row, column); // right
+
+        }
+
+    }
+
+    public static int findIslands(boolean[][] grid) {

```

Comment:
Make sure that your public method has a javadoc comment explaining what it returns (and in this case what it also does to the input grid)